tools.go

```diff
@@ -0,0 +1,5 @@
+package contrib
+
+import (
+	_ "github.com/google/addlicense"
```

Comment:
I added this file to keep `github.com/google/addlicense` in the `go.mod`. Without it, `go.mod` and `go.sum` always change after run `go mod tidy` which is the default behavior of GQLGen.